### PR TITLE
Adding SmartyStreets in the Geo section.

### PR DESCRIPTION
**Pull request by SunSparc**
The SmartyStreets US Reverse Geocoding API takes lat/long coordinates and transforms them into addresses. SmartyStreets does it super fast and super accurate.

### CODE REVIEW

**Comment by agamm** [comment]
@SunSparc  Can we put `https://www.smartystreets.com/` instead so it includes all the other sub products (they also seem to be related to geo) ?

**Comment by SunSparc** [comment]
@agamm, yes, certainly. That is a good idea. I will make that change.

File: README.md [hunk 1]
```diff
@@ -120,6 +120,7 @@ The definition of developer-first for this repo is:
 *Location services.*
 * [Radar](https://radar.io/) - Geofencing and geo APIs.
 * [mapbox](https://www.mapbox.com/) - Maps and locations products for devs.
+* [SmartyStreets](https://www.smartystreets.com/) - US Reverse Geocoding API.
```
Comment (agamm) [inline README.md:123]:
```suggestion
* [SmartyStreets](https://www.smartystreets.com/) - US Reverse Geocoding and Address Verification APIs.
```

**Review by agamm — CHANGES_REQUESTED** [review]
Better, as it encompasses more tools?

**Review by SunSparc — COMMENTED** [review]
Yes, that is a good change. I like that.

**Review by agamm — APPROVED** [review]
Thanks for the suggestion!

**Pull request merged**